mParticle-CompanyName/MPKitCompanyName.m

```diff
@@ -104,7 +107,7 @@ If your company SDK instance is available and is applicable (Please return nil i
 //         Please see MPKitExecStatus.h for all exec status codes
 //      */
 //
-//     MPKitExecStatus *execStatus = [[MPKitExecStatus alloc] initWithSDKCode:@(MPKitInstanceBranchMetrics) returnCode:MPKitReturnCodeSuccess];
+//     MPKitExecStatus *execStatus = [[MPKitExecStatus alloc] initWithSDKCode:@(MPKitInstanceCompanyName) returnCode:MPKitReturnCodeSuccess];
```

Comment:
What do you think about using `[[self class] kitCode]` rather than `@(MPKitInstanceCompanyName)`?